### DISCUSSION TITLE
Secondary list text on GTR4 (kinda urgent)

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#119] [Developer Feedback]
I cannot remember how many updates there was, but the second one messed up secondary list look, text should use **text_style.NONE** (keep scrolling) effect yet somehow someone put **text_style.WRAP** there and after several months nothing has changed, kindly please fix it. Thank you.

![photo_2023-03-25_11-44-36](https://user-images.githubusercontent.com/1674725/227712643-d5be8e7c-d80c-4cb7-a58f-c49b6e74df9f.jpg)
![photo_2023-03-25_11-44-38](https://user-images.githubusercontent.com/1674725/227712646-aae297ef-dbf5-4f00-a616-67f75874604f.jpg)
![photo_2023-03-25_12-27-37](https://user-images.githubusercontent.com/1674725/227714769-1e71b3a2-07ac-4334-988e-d4d178820f11.jpg)

From what I know problem exists on PL/RU langs.
@MyoungXUE @htoooth

**Comment by Xuemuyang** [comment]
Thank you for your feedback, I will feedback this issue to the relevant colleagues.

**Comment by MyoungXUE** [comment]
Hi, can you provide some code snippets to help us troubleshoot the problem？

**Reply by zdunex25** [reply]
Those screenshots are part of built-in System settings, like Settings> Screen or Preferences, just grab any GTR4 and change it's language to Polish or Russian and you will see.
Basically any line of text that should have a scrolling effect is just moving the part that is too long to the next line.
Does not happen on English tho.

**Reply by MyoungXUE** [reply]
Thank you for your cooperation, this is a bug, I will follow up on this issue and solve it as soon as possible.
